profiling/src/collections/string_table/wordpress_test_data.rs

```diff
@@ -0,0 +1,1064 @@
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+pub const WORDPRESS_STRINGS: [&str; 1059] = [
```

Comment:
A comment explaining where this came from, and how to modify it as needed, would be helpful to future maintainers.

Reply:
👍🏻 I'll move the comment from `test_wordpress` to the top of this file.